docs/templates/index.md

```diff
@@ -163,27 +171,53 @@ resource "coder_agent" "coder" {
 # that does not require root permissions. Note that /tmp may be mounted in tmpfs which
 # can lead to increased RAM usage. To avoid this, you can pre-install code-server inside
 # the Docker image or VM image.
+echo "Installing code-server..."
 curl -fsSL https://code-server.dev/install.sh | sh -s -- --method=standalone --prefix=/tmp/code-server --version 4.8.3
 
 # The & prevents the startup_script from blocking so the next commands can run.
 # The stdout and stderr of code-server is redirected to /tmp/code-server.log.
+echo "Starting code-server..."
 /tmp/code-server/bin/code-server --auth none --port 13337 >/tmp/code-server.log 2>&1 &
 
-# var.repo and var.dotfiles_uri is specified
-# elsewhere in the Terraform code as input
-# variables.
+# Notice: var.repo and var.dotfiles_uri are specified elsewhere in the Terraform
+# code as input variables.
+REPO=${var.repo}
+DOTFILES_URI=${var.dotfiles_uri}
```

Comment:
```suggestion
DOTFILES_URI=${data.coder_parameter.dotfiles_uri}
```
Should we suggest using the `coder_parameter` resource here as we do not support terraform variables anymore for workspace-level variables?

Reply:
I think that's a good idea, but I'd like to avoid making the change in this PR. Perhaps we should check if there's other places this correction needs to be done as well and do them all in one go?